src/libraries/System.Security.Cryptography.X509Certificates/tests/SignatureSupport.cs

```diff
@@ -0,0 +1,14 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Xml;
+using Xunit;
+
+namespace System.Security.Cryptography.X509Certificates.Tests
+{
+    public class SignatureSupport
+    {
+        public static bool SupportsRsaSha1Signatures { get; } =
```

Comment:
Let's rename it at this layer so it justifies suppressing DSA.

```suggestion
        public static bool SupportsX509Sha1Signatures { get; } =
```